src/engine.cpp

```diff
@@ -1,3 +1,48 @@
 #include "engine.h"
-
+#include <iostream>
+#include <string>
+#include <fstream>
+#include <random>
 // Create definitions of the RandomEngine class here
+// Methods:
+void RandomEngine::setLength(uint8_t l){
+    length = l;
+}
+uint8_t RandomEngine::getLength(){
+    return length;
+}
+void RandomEngine::setSymbol(bool s){
+    symb = s;
+}
+bool RandomEngine::getSymbolStatus(){
+    return symb;
+}
+std::string RandomEngine::getString(){
+    int mod=62;
+    if(getSymbolStatus()){
+        mod=72;
+    }
+    //Random number generation:
+    char seq[100];
```

Comment:
Setting up of seed must be done in the constructor itself. The problem here is if the this method is called multiple times, it will set seed every time. Here we should only generate the string only. 
This would mean making distribution and generator the member of the class itself.